internal/ls/signaturehelp.go

```diff
@@ -617,7 +620,7 @@ func getContainingArgumentInfo(node *ast.Node, sourceFile *ast.SourceFile, check
 	for n := node; !ast.IsSourceFile(n) && (isManuallyInvoked || !ast.IsBlock(n)); n = n.Parent {
 		// If the node is not a subspan of its parent, this is a big problem.
 		// There have been crashes that might be caused by this violation.
-		// Debug.assert(rangeContainsRange(n.parent, n), "Not a subspan", () => `Child: ${Debug.formatSyntaxKind(n.kind)}, parent: ${Debug.formatSyntaxKind(n.parent.kind)}`);
+		debug.Assert(RangeContainsRange(n.Parent.Loc, n.Loc), fmt.Sprintf("Not a subspan. Child: %s, parent: %s", n.KindString(), n.Parent.KindString()))
```

Comment:
Even with the build tag a lot of this will end up being executed eagerly; theoretically you could accept a func for these still and then they won't be called in the inlined empty functions so should be eliminated. Not clear how much that matters in this version of debugging though where we only get these in tests.

Reply:
Eh, I don't know why this one was a deferred message to begin with - it was just stringifying two `kind`s, nothing actually expensive. And it's the only assertion (thus far) using the deferred message details pattern, so I just eager'd it rather than adding a deferred-message-taking-overload. In an ideal world under a `release` tag, the whole `fmt.Sprintf` *should* get elided since its' result is unused, anyway. (Weather it actually does, I haven't checked in godbolt, but from what I've read on assertion libs in `go`, that's the expected codegen for something like this.)

Reply:
It'll still get executed, as far as I know. If this were a pure string concat, that would probably get elided.